ui/src/adapters/api/index.ts

```diff
@@ -21,32 +21,23 @@ interface APISuccessfulResponse<D> {
 
 export type APIResponse<D> = APISuccessfulResponse<D> | APIErrorResponse;
 
-export enum HTTPStatusError {
-  Aborted = 0,
-  Unauthorized = 401,
-  NotFound = 404,
-}
+export type HTTPStatusError = 0 | 400 | 401 | 404;
 
-export enum HTTPStatusSuccess {
-  Accepted = 202,
-  Created = 201,
-  NoContent = 204,
-  OK = 200,
-}
+export type HTTPStatusSuccess = 200 | 201 | 202 | 204;
```

Comment:
What do we need `HTTPStatusError` and `HTTPStatusSuccess` for?

Reply:
We need to properly type the usage, like in `status` for `APIError` and `ResponseError`.

Reply:
All this will be proposed for removal [here](https://polygonid.atlassian.net/browse/PID-488) so I don't really care much.